src/org/thoughtcrime/securesms/MediaPreviewActivity.java

```diff
@@ -230,13 +230,6 @@ private void initializeMedia() {
 
     // if you search for the place where the media are loaded, go to 'onCreateLoader'.
 
-    if (!isContentTypeSupported(initialMedia.type)) {
-      Log.w(TAG, "Unsupported media type sent to MediaPreviewActivity, finishing.");
-      Toast.makeText(getApplicationContext(), R.string.error, Toast.LENGTH_LONG).show();
-      finish();
-      return;
-    }
```

Comment:
As `isContentTypeSupported` takes a `Slide` now, and we don't have one here, I removed this check. It turned out not to work, anyway.